src/components/AppLayout/AppLayout.jsx

```diff
@@ -0,0 +1,15 @@
+import AppHeader from 'components/AppHeader/AppHeader.jsx';
+import AppFooter from 'components/AppFooter/AppFooter.jsx';
+import styles from './AppLayout.module.css';
+
+const AppLayout = ({ children }) => {
+    return (
+        <div className={styles.appLayout}>
+            <AppHeader />
+            <main className={styles.mainContent}>{children}</main>
```

Comment:
As you use react-router-dom upper than 6th vertion there is a component Outlet you can use instead of children. Please have a look